index.js

```diff
@@ -90,6 +93,11 @@ function stripLoc (doc, removeLocAtThisLevel) {
     delete doc.loc;
   }
 
+  if (doc.loc) {
```

Comment:
Will add a comment, the parser's loc fields include references to the query string and as your AST is deeper it becomes more overhead to ship/store the query